src/desktop/apps/consign/components/create_account/index.tsx

```diff
@@ -9,20 +9,24 @@ import { ModalType } from "@artsy/reaction/dist/Components/Authentication/Types"
 import { AuthIntent, ContextModule } from "@artsy/cohesion"
 
 interface CreateAccountProps {
+  contextPath: string
+  subject: string
   title: string
   type: ModalType
   updateAuthFormStateAndClearErrorAction: (type: ModalType) => void
 }
 
 export class CreateAccount extends React.Component<CreateAccountProps> {
   handleSubmit = (values, formikBag) => {
+    const { contextPath, subject, title, type } = this.props
+
     handleSubmit(
-      this.props.type,
+      type,
       {
-        copy: this.props.title,
+        copy: title,
         contextModule: ContextModule.consignSubmissionFlow,
         intent: AuthIntent.consign,
-        redirectTo: "/consign/submission",
+        redirectTo: `/consign/submission?contextPath=${contextPath}&subject=${subject}`,
```

Comment:
It might be nice to have a helper for building this URL in several places, similar to the related Reaction PR.

Reply:
I moved it into a getter on the class 👍